ors-benchmark/src/main/java/org/heigit/ors/coordinates_generator/generators/CoordinateGeneratorMatrix.java

```diff
@@ -200,37 +229,46 @@ private void shutdownExecutor(ExecutorService executor) {
                 executor.shutdownNow();
             }
         }
+
+        // Close HTTP client
+        try {
+            httpClient.close();
+        } catch (IOException e) {
+            LOGGER.error("Error closing HTTP client", e);
+        }
```

Comment:
Use autoclosable within the first try and catch?

Reply:
See comment above